extensions/notification/utils/config.js

```diff
@@ -0,0 +1,89 @@
+const constants = require('../constants');
+
+/**
+ * Transforms array of configurations to a Map
+ * @param  {Object[]} configs - Array of bucket notification configurations
+ * @return {Map} Map of bucket notification configurations
+ */
+function transform(configs) {
```

Comment:
`transform` seems too generic, the function name should describe somehow what the transform is about, like `configArrayToMap` for example